pyhap/accessory_driver.py

```diff
@@ -510,8 +528,10 @@ def stop(self):
         self.accessory_thread.join()
 
         logger.debug("Stopping mDNS advertising")
-        self.advertiser.unregister_service(self.mdns_service_info)
```

Comment:
`unregister_service` not necessary, since already included in `advertiser.close()`